lib/fact_check_email_handler.rb

```diff
@@ -34,12 +34,14 @@ def process_message(message)
 
   # &after_each_message: an optional block to call after processing each message
   def process(&after_each_message)
-    Mail.all(read_only: false, delete_after_find: true) do |message|
-      message.skip_deletion unless process_message(message)
-      begin
-        after_each_message.call(message) if after_each_message
-      rescue => e
-        GovukError.notify(e)
+    if ENV["RUN_FACT_CHECK_FETCHER"]
```

Comment:
I wonder if should we do `ENV.include?` here just in case someone thinks they can set the variable to `false` instead of just not setting it? I realise currently it's the same behaviour as before so not a big deal.

Reply:
I think it's worth using `ENV.include?`, I agree it's a bit of a duff test otherwise.